README.markdown

```diff
@@ -172,3 +172,47 @@ payload in querystring format.
 
 JavaScript and CSS are minified and combined. The files to be packaged and their orders are specified in `_includes/head.html` and <code>CssMinify.yml</code>. Preprocessing and options can be specified
 via `_plugins/jekyll_asset_pipeline.rb`.
+
+## Node Spellchecker
+
+This is a spellchecker functionality using [node-markdown-spellcheck](https://www.npmjs.com/package/markdown-spellcheck) node package.
+
+### Init
+```
+# (pull code updates before this)
+# install updated npm package dependencies!
+npm install
+```
+
+### Usage
+```
+# generates report of misspellings
+npm run spellcheck
```

Comment:
Can you please add 
`npm run spellcheck` to the "after_script" param in travis.yml?

This will cause it to run on every build ;)